sources/.nvmrc

```diff
@@ -1 +1 @@
-v16
+v20.10.0
```

Comment:
Drive-by: this matches the version in `.tool-versions`: https://github.com/gradle/actions/blob/main/sources/.tool-versions#L2